### PR TITLE
Make documentation neat

### PR DESCRIPTION
**Pull request by etylsarin**
As the actual version of documentation confused me a bit and I had to spend some time to find out how to set it up, these minor tweaks should fix it.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ grunt.initConfig({
 		options: {
 			format: require('eslint-tap')
 		},
-		target: ['file.js']
+		src: ['js/*.js']
```
Comment (sindresorhus) [inline readme.md]: This is incorrect.

**Comment by etylsarin** [comment]
Well, sorry @sindresorhus, don't want to waste your time, the only thing is that if you copy/paste your example, the plugin throw "Could not find any files to validate". Just wanted to make it easier for anyone like me using it from scratch.

**Comment by sindresorhus** [comment]
You of course need to change the path to where the files are. Throwing is actually a good thing as most users just copy paste and wouldn't get any notification if they used your version and had JS files in a different directory.

**Comment by etylsarin** [comment]
And does the "target" property do anything? if it does not, it shouldn't be in the example imho. And some mention of the standard "src" property would be helpful as well. Just saying. Feel free to ignore me. :)

**Comment by sindresorhus** [comment]
It's a `target`. Multi task plugins, like most grunt plugins are, have targets. `src` is something different and it would be confusing to name your target `src`.

**Comment by etylsarin** [comment]
Oh ok, I see now how you mean it. Sorry for being so lame today.

**Comment by sindresorhus** [comment]
No worries. The grunt config format is very ambiguous...

**Pull request closed without merging**